rust/protocol/src/storage/traits.rs

```diff
@@ -94,8 +146,10 @@ pub trait SessionStore {
     ) -> Result<()>;
 }
 
+/// Interface for storing sender keys for other users, allowing multiple keys per user.
```

Comment:
Let's say "sender key records", since they're more than just keys. (They're sessions of their own, just not Double Ratchet sessions.)

Reply:
This was a super helpful remark! Thanks!